docs/UserGuide.md

```diff
@@ -18,76 +18,135 @@ Here is a **quick** overview of your internship journey with LinkMeIn:
+   **examples** to help you better understand how to use the feature.
+2. To quickly refer to the command format of a feature, you can refer to the [Command Summary Section](#command-summary)
+where all the commands are listed out in a table format.
+3. If you face any issue, you can refer to the [FAQ Section](#faq) to find out if your issue has been addressed before.
+
+
 # Table of Contents
 
 * TOC 
 {:toc}
 
 # Quick start
+In this section, you will learn how to set up LinkMeIn in your computer. 
+
+**Step 1: Verify Java Version:**
 
-1. Make sure that you have **Java 11 or above** installed in your Computer.
-  - [How to check if you have Java installed?](#faq)
-  - [How to check if you have Java 11 or above?](#faq)
-2. Download the latest jar file from [here](https://github.com/AY2324S1-CS2103T-T17-2/tp/releases/tag/v1.2.0)
-3. Copy the file to the folder you want to use as the home folder for your LinkMeIn.
-* Create a new folder (you can name it LinkMeIn) where you would like to house the application
-    (e.g., Desktop, Documents, etc.)
-* Move the downloaded jar file as shown:
-  * For Windows users 
-  * For Mac users
-4. Launch LinkMeIn
-* For Windows users 
-  * Double-click on the `LinkMeIn.jar` file to launch the application
-
-* For Mac users
-  * Open up your terminal (Command +Space > type Terminal > Enter)
-  * Navigate to the folder where you have placed the jar file with the `cd` command 
-  * For example if you have placed the jar file in the Documents folder, type `cd Documents` and press Enter
+1. Make sure that you have **Java 11 or above** installed in your Computer. Click 
+[here](#1-how-do-i-check-if-i-have-java-11-installed-) if you are unsure of how to check if you have Java 11 installed.
+
+**Step 2: Download LinkMeIn:**
+1. With Java 11 installed, you can now download our latest jar file from [here](https://github.com/AY2324S1-CS2103T-T17-2/tp/releases/tag/v1.2.0)
+2. Click on the `LinkMeIn.jar` file to download it.
+
+![githubRelease.png](images/quick-start/githubRelease.png)
+
+**Step 3: Choose a home folder for LinkMeIn:**
+1. Decide on a folder to use as the home folder for LinkMeIn. This folder will be used to store the data of LinkMeIn.
+2. Create a new folder (you can name it "LinkMeIn") in a location of your choice, 
+such as your Desktop or Documents folder
+3. Move the downloaded jar file to the folder you have created in the previous step.
+
+**Step 4: Launch LinkMeIn:**
+Depending on your operating system, do one of the following to launch LinkMeIn:
+1. For Windows users: 
+   - Double-click on the `LinkMeIn.jar` file to launch the application
+2. For Mac users:
+   * Open up your terminal. If you are unsure of how to do so, you can refer to [here]().
+   * Navigate to the folder where you have placed the jar file with the `cd` command 
+   * For example if you have placed the jar file in the Documents folder, type `cd Documents` and press Enter
+   * Type `java -jar LinkMeIn.jar` and press Enter
```

Comment:
Missing full stops for last 3 bullet points.
Missing link for Mac users on the here